user_docs/en/userGuide.t2t

```diff
@@ -2866,6 +2876,13 @@ Specifically, following issues can be solved by running this tool:
 ++ Reload plugins ++[ReloadPlugins]
 This item, once activated, reloads app modules and global plugins without restarting NVDA, which can be useful for developers.
 
+The following NVDA key commands may also be useful:
+%kc:beginInclude
+|| Name | key | Description |
+| Reload plugins | ``NVDA+control+f3`` | Reloads NVDA's global plugins and app modules. |
+| Report loaded app module and executable | ``NVDA+control+f1`` | Report the name of the app module, if any, and the name of the executable associated to the active application which has the keyboard focus. |

```

Comment:
Re-listing this one in case you missed it. ("to" change to "with")
Also, is "active" necessary when you say "which has the keyboard focus"?
```suggestion
| Report loaded app module and executable | ``NVDA+control+f1`` | Report the name of the app module, if any, and the name of the executable associated with the application which has the keyboard focus. |
```